openff/toolkit/tests/test_parameters.py

```diff
@@ -1403,6 +1403,9 @@ def test_different_defaults_03_04(self):
         bh2 = BondHandler(version=0.4)
         assert bh2.fractional_bondorder_method == "AM1-Wiberg"
         assert bh2.potential == "(k/2)*(r-length)^2"
+        bh3 = BondHandler(version=0.3, fractional_bondorder_method="AM1-Wiberg")
+        assert bh3.fractional_bondorder_method == "AM1-Wiberg"
```

Comment:
```suggestion
        assert bh3.fractional_bondorder_method == "AM1-Wiberg"
        assert bh3.fractional_bondorder_interpolation == "linear"
```
Just to be safe - it wasn't checked previously